node-runner-cli/commands/dockercommand.py

```diff
@@ -185,12 +185,12 @@ def start(args):
 
 
 @dockercommand([
-    argument("-f", "--configfile",
-             help="Path to config file. This file is generated by running 'babylonnode docker config'"
-                  f"The default value is `{Helpers.get_default_node_config_dir()}/config.yaml` if not provided",
-             default=f"{Helpers.get_default_node_config_dir()}/config.yaml",
+    argument("-f", "--composefile",
```

Comment:
Why we changing the config file to compose file?

Reply:
oh wait, it was a change committed by @kofrdx